bedrock/mozorg/templates/mozorg/about/includes/m24/community.html

```diff
@@ -4,7 +4,7 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-<div class="m24-t-purple">
+<div class="m24-t-purple m24-t-transition-next">
```

Comment:
Please apply the transition HTML changes to the about page as well.

Reply:
I thought I did... current Figma shows the only pixel effect on the about page is the footer, so this class adds the needed padding to the last section.

Though one thing I considered is that if the footer ALWAYS has the pixel effect it means the last section on EVERY page will need adequate padding for it, and I'm not sure how best to approach that if we also need to be able to change the background of the bottommost section of pages. Maybe a global :last-child rule? I wish we had a previous-sibling selector... hmm

There are a few other things on the about page that this batch of changes does break, mainly some refactoring of the showcase section, but I didn't want to collide with other WIP on that page so I left it alone. But it makes sense for any sweeping changes in one PR to implement them in one sweep as well, it just might make a tricky merge conflict.

Reply:
Oh, I think when I wrote this I thought the transition changes explained why there was no top or bottom padding on the showcases anymore. Anyway, please fix the about page. Which you are already doing :)